src/scss/index.scss

```diff
@@ -1 +1,154 @@
+$primary: #67927a;
+$secondary: #eae6e5;
+$dark-color: #000;
+$secondary-light: #eae6e5c4;
+$primary-light: #67927ae9;
+$box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.16);
```

Comment:
You are not importing Bootstrap!

`@import "../../node_modules/bootstrap/scss/bootstrap";`

Reply:
started all over again: bootstrap is now correctly installed